src/python-bindings/htcondor2/remote_param.cpp

```diff
@@ -0,0 +1,242 @@
+bool
+start_config_command( int cmd, ReliSock & sock, const ClassAd & location ) {
+    std::string address;
+    if(! location.EvaluateAttrString( ATTR_MY_ADDRESS, address )) {
+        // This was HTCondorValueError in version 1.
+        PyErr_SetString( PyExc_ValueError, "Address not available in location ClassAd." );
+        return false;
+    }
+
+    // Not sure we actually need to make this copy, but version 1 did.
+    ClassAd copy;
+    copy.CopyFrom(location);
+
+    Daemon d( &copy, DT_GENERIC, NULL );
+
+    CondorError errorStack;
```

Comment:
Since the ``errorStack`` is presumably being populated upon failure, I feel like we should do something with the information. Not sure if it warrants going into the Exception message, but perhaps a ``dprintf`` incase debugging is set up. Not sure. Note: I think it would be nice but am willing to do nothing with this information if we decide to ignore keep it as is.

Reply:
It's easy enough to add a dprintf().  We should probably think about whether or not the Python bindings C++ code should ever dprintf(), or if we want some sort of consistent rule about it, but we can do that later; it's easy enough to remove dprintf()s later, too.